src/components/Input/PaymailAutocomplete.tsx

```diff
@@ -57,31 +58,32 @@ export const PaymailAutocomplete = forwardRef<HTMLInputElement, PaymailAutocompl
 PaymailAutocomplete.displayName = 'PaymailAutocomplete';
 
 const Listbox = styled.ul`
-  width: 100%,
-  margin: 0,
-  padding: 0,
-  zIndex: 1,
-  position: absolute,
-  listStyle: none,
-  backgroundColor: ${colors.primaryBackground},
-  overflowX: hidden,
-  overflowY: auto,
-  scrollbarWidth: thin,
-  scrollbarColor: ${colors.secondaryBackground} ${colors.primaryBackground},
-  boxShadow: 0 2px 4px rgba(0,0,0,.25),
-  borderRadius: ${sizes(1)},
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    z-index: 1;
+    position: absolute;
+    list-style: none;
+    background-color: ${colors.thirdBackground};
+    overflow-x: hidden;
+    overflow-y: auto;
+    scrollbar-width: thin;
+    scrollbar-color: ${colors.secondaryBackground} ${colors.primaryBackground};
+    box-shadow: 0 2px 4px rgba(0, 0, 0, .25);
+    border-radius: ${sizes(1)};
 
-  maxHeight: 200,
-  border: 1px solid rgba(0,0,0,.25),
-  & li.Mui-focused: {
-    backgroundColor: #4a8df6,
-    color: white,
-    cursor: pointer,
-  },
-  & li:active: {
-    backgroundColor: #2977f5,
-    color: white,
-  },
-  & li: {
-    padding: ${sizes(1)},
-  },`
+    max-height: 200px;
+    border: 1px solid rgba(227, 227, 227, 0.25);
+
+    & li.Mui-focused {
+        background-color: ${colors.textHover};
+        color: white;
+        cursor: pointer;
+    }
+
+,
+& li {
+    background-color: ${colors.thirdBackground};
+    padding: ${sizes(1)};
+},
```

Comment:
Is it ok, that it has different indentation?

Reply:
No idea. I mean. There is no prettier in the project so... 

Anyway i fixed it manually.